modules/460-log-shipper/hooks/internal/vector/source/file.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/deckhouse/deckhouse/modules/460-log-shipper/apis/v1alpha1"
 )
 
+var _ apis.LogSource = (*File)(nil)
```

Comment:
what is this line for?
In the composer.go file we have `newLogSource` which calls `source.NewFile`. if interface `apis.LogSource ` is not implemented - it wouldn't compile at that place. We don't need to check it one more time

Reply:
> if interface apis.LogSource is not implemented - it wouldn't compile at that place.
Yeah, this is kind of a safety measurement. The behavior of the `newLogSource` can change in the future, but this line's only purpose is to check the interface. I did it just to be hundred percent sure everything is fine with the code.